changelog.d/1-api-changes/WPB-4668-disable-defederation

```diff
@@ -0,0 +1 @@
+Disable de-federation (to avoid running into a scalability issue).
```

Comment:
I think we should also delete other changelog entries which talk about defederation

Reply:
i would have left that as an exercise for the release curator, but fair, will do!